src/components/message/Message.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import AppCard from '~/components/app-card/AppCard'
+import UserProfileInfo from '~/components/user-profile-info/UserProfileInfo'
+
+import { MessageInterface } from '~/types/chat/message/message.interface'
```

Comment:
Try to do the correct export types.

Create a new file "message.index.ts" and add `export * from "~/types/chat/message/message.interface"` then add in index.ts file `export * from "~/types/chat/message/message.index".`
And after that, you will receive here `import { MessageInterface } from '~/types'`

Reply:
> Try to do the correct export types.
> 
> Create a new file "message.index.ts" and add `export * from "~/types/chat/message/message.interface"` then add in index.ts file `export * from "~/types/chat/message/message.index".` And after that, you will receive here `import { MessageInterface } from '~/types'`

thank you for your advice, fixed it